ml-algorithms/src/test/java/org/opensearch/ml/engine/algorithms/remote/AwsConnectorExecutorTest.java

```diff
@@ -122,7 +122,7 @@ public void executePredict_RemoteInferenceInput_EmptyIpAddress() {
             .credential(credential)
             .actions(Arrays.asList(predictAction))
             .build();
-        connector.decrypt((c) -> encryptor.decrypt(c));
+        connector.decrypt((c, tenantId) -> encryptor.decrypt(c, null), null);
```

Comment:
This is starting to strain readability.  

Should the last part of that be `encryptor.decrypt(c, tenantId), null` ?

If so I think this is the same thing and more readable: `encryptor::decrypt, null`

If not, deserves a comment explaining why we're throwing away tenant ID, probably best put above a constant bifunction you can throw into this method to replace `(c, tenantId) -> encryptor.decrypt(c, null)`

Reply:
Here,  the `decrypt` method of the connector is called with a lambda function and null as the arguments.

1. This lambda function is an implementation of the BiFunction<String, String, String> interface.
2. It takes two parameters: c (which represents the encrypted text) and tenantId (which is not used in this context).
3. Inside the lambda function, encryptor.decrypt(c, null) is called.c is passed as the encryptedText.
null is passed as the tenantId to the encryptor.decrypt method.